share/functions/funced.fish

```diff
@@ -71,6 +71,11 @@ function funced --description 'Edit function definition'
 
     if not functions -q -- $funcname
         echo $init >$tmpname
+    else if string match --invert --quiet --regex '^(?:n/a|not-autoloaded|autoloaded)$' -- (functions --details --verbose -- $funcname)[2]
```

Comment:
Probably `not string match` would be a little bit more idiomatic than `string match --invert`.

It's a shame that we don't have an easier / more robust way of figuring out that a function was copied from somewhere.

Reply:
The line later had that `--invert` so I had kept it. changed it to `not`.